ios/RNNativePlayground/RNOldHotelsOptions.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import <RNKiwiMobile/RNKiwiMobile.h>
+
+@interface RNOldHotelsOptions : UIViewController <RNKiwiOptions>
+
+- (instancetype)initWithDimensions:(NSDictionary *)dimensions andNavigationMode:(NSString *)lastNavigationMode;
```

Comment:
Is this still good? like... can we just have something like `initWithParams` and pass both? Just saying.

Reply:
Refactored